plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -311,6 +304,7 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
 
       private fun finishAnimation(animation: Animator?, finishStatus: AnimationFinishStatus) {
         (animation as? CameraAnimator<*>)?.apply {
+          // add current animator to queue-set if was not present
```

Comment:
It seems we're removing animation here and not adding, comment is a bit confusing then